conf/openmetadata.yaml

```diff
@@ -326,3 +326,5 @@ web:
 changeEventConfig:
   omUri: ${OM_URI:- "http://localhost:8585"} #openmetadata in om uri for eg http://localhost:8585
 
```

Comment:
can we make this optional in Configuration. i.e if this section doesn't exists server should continue to start